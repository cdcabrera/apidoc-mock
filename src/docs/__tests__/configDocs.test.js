const { parse, path, getGroup, method } = require('../configDocs');

describe('ConfigDocs', () => {
  it('should have specific defined properties', () => {
    expect(parse).toBeDefined();
    expect(path).toBeDefined();
    expect(getGroup).toBeDefined();
    expect(method).toBeDefined();
  });

  it('should parse custom settings and return an object', () => {
    const content = '{DelayResponse} 3000';
    const source = '@apiMock {DelayResponse} 3000';
    const defaultGroup = undefined;

    expect(parse(content, source, defaultGroup)).toMatchSnapshot();
  });
});
