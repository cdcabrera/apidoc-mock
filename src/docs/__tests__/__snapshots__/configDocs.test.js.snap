// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfigDocs should parse custom settings and return an object 1`] = `
{
  "delayResponse": "3000",
}
`;
